Classes/Hooks/Datahandler/CommandMapAfterFinishHook.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace  B13\Container\Hooks\Datahandler;
+
+/*
+ * This file is part of TYPO3 CMS-based extension "container" by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+use B13\Container\Domain\Factory\Database;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\DataHandling\DataHandler;
+use B13\Container\Domain\Factory\ContainerFactory;
+
+class CommandMapAfterFinishHook
+{
+    /**
+     * @var Database
+     */
+    protected $database = null;
+
+    /**
+     * @param ContainerFactory|null $containerFactory
+     * @param Database|null $database
+     */
+    public function __construct(Database $database = null)
+    {
+        $this->database = $database ?? GeneralUtility::makeInstance(Database::class);
+    }
+
+    /**
+     * @param DataHandler $dataHandler
+     */
+    public function processCmdmap_afterFinish(DataHandler $dataHandler): void
+    {
+        $cmdmap = $dataHandler->cmdmap;
+        $copyMappingArray_merged = $dataHandler->copyMappingArray_merged;
+
+        foreach ($cmdmap as $table => $_) {
+            foreach ($cmdmap[$table] as $id => $incomingCmdArray) {
+                if (!is_array($incomingCmdArray)) {
+                    continue;
+                }
+                if ($table !== 'tt_content') {
+                    continue;
+                }
```

Comment:
This condition should be moved to the outer loop

Reply:
fixed